maintenance/shfmt.sh

```diff
@@ -1,16 +1,11 @@
 #!/usr/bin/env bash
-set -Eeuo pipefail
 
 shfmt_args=(
-    --indent 4
-    --keep-padding      # keep column alignment paddings
-    --space-redirects   # redirect operators will be followed by a space
-    --binary-next-line  # binary ops like && and | may start a line
-    --case-indent       # switch cases will be indented
+    -i=4
+    -kp      # keep column alignment paddings
+    -sr   # redirect operators will be followed by a space
+    -bn  # binary ops like && and | may start a line
+    -ci       # switch cases will be indented
```

Comment:
The long args work with the more recent version as you saw ;)

Reply:
If that's ok with you i'm happy to keep a version that works on my system ^^

Reply:
Yeah alright, if that's more portable that's fine with me !